src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -1369,7 +1373,12 @@ private void AddComponentStats(StringBuilder log)
 
             lock (this.peerLock)
             {
-                if (this.isIbd) log.AppendLine("IBD Stage");
+                long currentTime = DateTimeProvider.Default.GetTime();
```

Comment:
This should use the an injected provider

Reply:
This mimics `InitialBlockDownloadState` which used the default provider for the same calculation:

![image](https://user-images.githubusercontent.com/29645989/61099821-fe54af80-a4a6-11e9-90d2-48232441f9f7.png)